cypress/e2e/example.cy.js

```diff
@@ -1,51 +1,69 @@
-  const user = 'Harry Potter';
-  const accountNumber = '1004';
-
-  before(() => {
-    cy.visit('/');
-  });
+  let balance;
+  const fullName = 'Hermoine Granger';
+  const accountNumber = '1001';
+  const currency = 'Dollar';
+
+  const depositAmount = '200';
+  const withdrawAmount = '100';
+
+  const depositSuccessMessage = 'Deposit Successful';
+  const withdrawlSuccessMessage = 'Transaction successful';
 
-  it('should provide the ability to work with bank account', () => {
+  beforeEach(() => cy
+    .visit('baseUrl'));
+  it('should allow wizzard to login and view transactions', () => {
     cy.contains('.btn', 'Customer Login').click();
-    cy.get('[name="userSelect"]').select(user);
-    cy.contains('.btn', 'Login').click();
-
-    cy.contains('[ng-hide="noAccount"]', 'Account Number')
-      .contains('strong', accountNumber)
-      .should('be.visible');
-    cy.contains('[ng-hide="noAccount"]', 'Balance')
-      .contains('strong', '0')
-      .should('be.visible');
-    cy.contains('.ng-binding', 'Dollar')
-      .should('be.visible');
-
-    cy.get('[ng-click="deposit()"]').click();
-    cy.get('[placeholder="amount"]').type(depositAmount);
-    cy.contains('[type="submit"]', 'Deposit').click();
-
-    cy.get('[ng-show="message"]')
-      .should('contain', 'Deposit Successful');
-    cy.contains('[ng-hide="noAccount"]', 'Balance')
-      .contains('strong', depositAmount)
-      .should('be.visible');
-
-    cy.get('[ng-click="withdrawl()"]').click();
-    cy.contains('[type="submit"]', 'Withdraw')
-      .should('be.visible');
-    cy.get('[placeholder="amount"]').type(withdrawAmount);
-    cy.contains('[type="submit"]', 'Withdraw').click();
-
-    cy.get('[ng-show="message"]')
-      .should('contain', 'Transaction successful');
-    cy.contains('[ng-hide="noAccount"]', 'Balance')
-      .contains('strong', balance)
-      .should('be.visible');
+    cy.get('#userSelect').select(fullName);
+    cy.get('button[type="submit"]').click();
+    cy.get('#accountSelect').should('contain', accountNumber);
+    cy.get('.borderM > :nth-child(3) > :nth-child(2)')
+      .invoke('text').then((text) => {
+        cy.log(`${text}`);
+
+        balance = text;
+        const balanceAfterDeposit = (+balance + +depositAmount)
+          .toString();
+        const balanceAfterWithdrawl = (+balanceAfterDeposit - +withdrawAmount)
+          .toString();
+
+        cy.get('.borderM > :nth-child(3) > :nth-child(3)')
+          .should('contain', currency);
+        cy.contains('.btn', 'Deposit ').click();
+        cy.get('[placeholder="amount"]').type(`${depositAmount}{enter}`);
+        cy.contains('.ng-binding', depositSuccessMessage)
+          .should('exist').and('be.visible');
+        cy.get('.borderM > :nth-child(3)')
+          .should('contain', balanceAfterDeposit);
+
+        cy.contains('.btn', 'Transactions ').click();
+        cy.contains('.btn', 'Back')
+          .click();
+
+        cy.contains('.btn', 'Withdrawl').click();
```

Comment:
There is a typo in the button text 'Withdrawl'. It should be 'Withdrawal' to match the correct spelling.